internal/entryhuman/entry.go

```diff
@@ -170,7 +170,7 @@ func isTTY(w io.Writer) bool {
 }
 
 func shouldColor(w io.Writer) bool {
-	return isTTY(w) || os.Getenv("FORCE_COLOR") != ""
+	return isTTY(w) || os.Getenv("TERM") != ""
```

Comment:
Just saw this and thought I'd give my two cents.

Using `TERM` could be problematic since now there'd be no way to disable colors. There are terminals like `dumb` and `xterm` (at least) that would not appreciate it 😄.

Whilst I haven't seen any form of standard for `FORCE_COLOR`, there are cases where it's used, [Node.js, for instance, uses it in a weird way](https://runebook.dev/en/docs/node/cli?page=2).

As far as cli color standards goes, [`NO_COLOR`](https://no-color.org/) does exist and is quite widely adopted. However, it's the inverse of what we want to do here. I think it'd be important to obey this flag to have at least some method of disabling colors (esp. with the TERM != "" check).

Reply:
We already suffer from that sending colors to terminals like dumb and xterm. If you have a dumb terminal connected, then we'll be outputting colors anyway as we'll detect a TTY and we treat all TTYs the same, even if `TERM=''`. Now, if you have TERM=dumb but then you also don't have a tty connected perhaps you're in a shell and trying to write logs to just a file where you don't want tty output but in that case it's really easy to just throw in `TERM=` to disable colors and means we don't have to invent a new variable.

Though it's ofc very non standard.

Yea good memory, node.js's use is bizarre.

I've just been really annoyed at having to put `$FORCE_COLOR` in all my CI scripts. I practically always want color, that's the whole reason I use sloghuman. I'm not 100% convinced about this PR either, I'll close for now.

Reply:
nvm i was wrong, looks like fatih/color does sort of check $TERM

https://github.com/fatih/color/blob/dbae876e45b9c53e464d101d63f4758c9f0bd8e8/color.go#L22